ios/RCTAgora/Base/BeanCovertor.swift

```diff
@@ -123,9 +123,9 @@ func mapToTranscodingUser(_ map: [String: Any]) -> AgoraLiveTranscodingUser {
 
 func mapToColor(_ map: [String: Any]) -> UIColor {
     return UIColor(
-            red: CGFloat((map["red"] as! NSNumber).intValue),
-            green: CGFloat((map["green"] as! NSNumber).intValue),
-            blue: CGFloat((map["blue"] as! NSNumber).intValue),
+            red: (map["red"] as! CGFloat) / 255,
```

Comment:
Can int value cast to CGFloat?
I think it should cast to NSNumber.
`CGFloat((map["red"] as! NSNumber).floatValue / 255)`

Reply:
NSNumber and CGFloat are interchangeable through casting since swift 2

for example this is valid:

```swift
let c1 = CGFloat(12.3)
let num = c1 as NSNumber
let c2 = num as CGFloat
```

Reply:
@LichKing-2234 any thoughts on this?